webpack/webpack-plugin-utilities/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@rushstack/webpack-plugin-utilities",
+  "version": "0.0.1",
+  "description": "This plugin sets the webpack public path at runtime.",
+  "main": "lib/index.js",
+  "typings": "dist/webpack-plugin-utilities.d.ts",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/rushstack.git",
+    "directory": "webpack/webpack-plugin-utilities"
+  },
+  "scripts": {
+    "build": "heft build --clean",
+    "_phase:build": "heft build --clean"
+  },
+  "peerDependencies": {
+    "@types/webpack": "^4.39.8",
```

Comment:
The name of this needs to match the name in devDependencies, otherwise it isn't available in published packages.